tests/gis_tests/geogapp/models.py

```diff
@@ -12,7 +18,7 @@ def __str__(self):
 
 
 class City(NamedModel):
-    point = models.PointField(geography=True)
+    point = models.PointField(geography=True, unique=supports_unique_geography)
```

Comment:
I would added a new model and use `Meta.required_db_features` instead:
```python
class CityUnique(NamedModel):
    point = models.PointField(geography=True, unique=True)

    class Meta:
        required_db_features = {
            "supports_geography",
            "supports_geometry_field_unique_index",
        }
```

Reply:
Nice, I started down that path and then saw new fixture entries would be necessary so I'll give that a go 👍